### DISCUSSION TITLE
Labo 2 - Drop shape

### DISCUSSION THREAD

**Discussion by Greenkiller21** [web-classroom/web-classroom.github.io#8] [Q&A : Pratique / Labos] [answered]
Imaginons une situation ou on se place avec la souris tout à droite de l'écran et que l'on ne bouge plus jusqu'à que la pièce touche le fond de la grille. Lorsque la pièce atteint le fond, doit-on faire en sorte que la nouvelle pièce du joueur soit placée à l'endroit où la souris est placé (même si celle-ci n'a pas encore changé de colonne) ou doit on laisser le comportment qu'il y'a actuellement (c'est-à-dire que la pièce soit placée dans la colonne du milieu) ?
Merci

**Comment by OlivierLmr** [comment, marked as answer]
Très bonne question !
On peut garder le comportement actuel : une nouvelle pièce est toujours placée en haut au centre, et elle n'est changée de colonne que lorsque la souris bouge.
Ceci dit, je ne pense pas qu'il existe un test pour cela, et vous pouvez vous considérer libres des choix que vous faites pour tout aspect non-mentionné dans l'énoncé (tant que les tests passent, bien sûr). Si vous trouvez plus pertinent de placer la pièce à la même colonne que celle qui vient d'être posée, libre à vous !
